cfme/tests/infrastructure/test_host.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+
+import pytest
+import random
+
+from utils.providers import setup_a_provider
+from cfme.web_ui import Quadicon, fill
+from cfme.infrastructure.host import credential_form
+import cfme.fixtures.pytest_selenium as sel
+import cfme.web_ui.toolbar as tb
+
+
+@pytest.fixture(scope="module")
+def provider():
+    return setup_a_provider(prov_class="infra", prov_type="virtualcenter")
+
+
+# Test to automate BZ 1201092
+def test_multiple_host_creds(provider):
+    pytest.sel.force_navigate('infrastructure_hosts')
+
+    quads = Quadicon.all("host", this_page=True)
+    for quad in quads:
+            sel.check(quad.checkbox())
+    tb.select("Configuration", "Edit Selected items")
+
+    cfme_host = random.choice(provider.get_yaml_data()["hosts"])
+
+    fill(credential_form, {'default_principal': 'root',
+        'default_secret': 'dog8code',
```

Comment:
Oh yes,that is on my to do list.I was trying to get something else to work,so forgot to remove the host creds.